migration/sql-files/002-tracker-items.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE ONLY tracker_items
```

Comment:
This is how I tested.

- Check out new code
- Migrate
- Run tests
- Check out old code
- Run tests

All tests passed in both new and old code.

If there is no other issues, please merge it.